### PR TITLE
{bio}[GCC/8.2.0-2.31.1] KMC v3.1.1 w/ Python 3.7.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3105.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/b3d4e5baa840cc91cb6fb1a1fc772a45 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso-1 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/0125a502d240c44754b853d41d85ae23 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 11 out of 11 (1 easyconfigs in this PR)
node2416.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d854fd34d8c5a96fda50658e7a887103 for a full test report.

File: easybuild/easyconfigs/k/KMC/KMC-3.1.1-GCC-8.2.0-2.31.1-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+checksums = ['d7cdf37d90a07d1a432b7427436f962914b5f63a1b6dbb9a116609a1c64d1324']
+
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.11'),
+    ('Python', '3.7.2'),
+]
+
+# Makefile does static linking with libc.a, libpthread.a, libm.a
+osdependencies = ['glibc-static']
```
Comment (akesandgren) [inline easybuild/easyconfigs/k/KMC/KMC-3.1.1-GCC-8.2.0-2.31.1-Python-3.7.2.eb]: Add libc6-dev for Debian/Ubuntu

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2405.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/6ab5107adc61b0a9cec316c67f6ca755 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/00d15b665162342cd048d87d1586270d for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**